src/platform/baytrail/platform.c

```diff
@@ -241,6 +241,20 @@ int platform_init(struct sof *sof)
 	shim_write(SHIM_PIMRH, shim_read(SHIM_PIMRH) | 0x00000700);
 #endif
 
+	/* Reset M/N SSP clock dividers */
+	shim_write(SHIM_SSP0_DIVL, 1);
+	shim_write(SHIM_SSP0_DIVH, 0x80000001);
+	shim_write(SHIM_SSP1_DIVL, 1);
+	shim_write(SHIM_SSP1_DIVH, 0x80000001);
+	shim_write(SHIM_SSP2_DIVL, 1);
+	shim_write(SHIM_SSP2_DIVH, 0x80000001);
+	shim_write(SHIM_SSP3_DIVL, 1);
+	shim_write(SHIM_SSP3_DIVH, 0x80000001);
+	shim_write(SHIM_SSP4_DIVL, 1);
+	shim_write(SHIM_SSP4_DIVH, 0x80000001);
+	shim_write(SHIM_SSP5_DIVL, 1);
+	shim_write(SHIM_SSP5_DIVH, 0x80000001);
+
```

Comment:
Do we need to put these part in the following `#if defined CONFIG_CHERRYTRAIL`?
We only have ssp0, ssp1, ssp2 for BYT.

Reply:
I'll move those inside the block, does that mean the shim window is actually 0x100 for baytrail and 0x118 for cht only? This affects another PR then for me. (https://github.com/thesofproject/linux/pull/1578)

Reply:
@cujomalainey Not sure, maybe you can read or write, but the result may be invalid.